docs/conf.py

```diff
@@ -278,8 +278,8 @@
 # (source start file, target name, title,
 #  author, documentclass [howto, manual, or own class]).
 latex_documents = [
-    (master_doc, 'Raiden.tex', u'Raiden Network Documentation',
-     u'Raiden Project', 'manual'),
+    (master_doc, 'Raiden.tex', 'Raiden Network Documentation',
```

Comment:
Since you are touching this, split each one in its own line or if they fit in one line put them in one line